tests/Client/RequestTest.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+namespace ekinhbayar\GitAmpTests\Client;
+
+use Amp\Artax\Client;
+use ekinhbayar\GitAmp\Client\GitAmp;
+use ekinhbayar\GitAmp\Events\Factory;
+use PHPUnit\Framework\TestCase;
+
+class RequestTest extends TestCase
+{
+    private $gitamp;
+
+    public function setUp()
+    {
+        $this->gitamp = new GitAmp(new Client, new \ekinhbayar\GitAmp\Github\Token('token'), new Factory());
+    }
+
+    public function tearDown()
+    {
+        $this->gitamp = null;
```

Comment:
Don't think this one is needed as it's getting overwritten anyway in the setUp method calls